File: Célia_LingProgram/src /Lista03/Exer08_Votacao.java
package Lista03;

import java.util.Random;

public class Exer08_Votacao {
    public static void main(String[] args) {
        System.out.println("Pesquisa SABOR FAVORITO!\n"
                + "Qual o seu sabor favorito de pizza? \n"
                + "       Sabores  | Código \n"
                + "     Mussarela  |    5   \n"
                + "     Calabresa  |   25   \n"
                + "Quatro Queijos  |   50   \n");

        Random aleatorio = new Random();
        Integer contador = 1;
        Integer M5 = 0;
        Integer C25 = 0;
        Integer Q50 = 0;

        while (contador <= 10) {
            Integer sorteado = aleatorio.nextInt(3);

            switch (sorteado) {
                case 0:
                    M5++;
                    break;
                case 1:
                    C25++;
                    break;
                case 2:
                    Q50++;
                    break;
            }

            contador++;
        }

        System.out.print("_________________________\n"
                + "   -----RESULTADO-----\n"
                + "O sabor favorito é ");
        if (M5 > C25 && M5 > Q50) {
            System.out.println("Mussarela");
        } else if (C25 > M5 && C25 > Q50) {
            System.out.println("Calabresa");
        } else {
            System.out.println("Quatro Queijos");
        }
        System.out.println("Mussarela: " + M5);
        System.out.println("Calabresa: " + C25);
        System.out.println("Quatro Queijos: " + Q50);
    }
}

File: Célia_LingProgram/JavaFacul/src/Lista03/Exer08_Votacao.kt
package Lista03

import kotlin.random.Random

//
//Foi feita uma pesquisa para saber qual o sabor de pizza favorito entre os alunos da
//Faculdade. Para votar no sabor mussarela, deve-se digitar o valor 5, Para votar no sabor calabresa,
// deve-se digitar o valor 25, Para votar no sabor quatro queijos, deve-se digitar o valor 50.
// Elabore um programa em Java que leia os votos de 10 alunos diferentes
//(sem usar vetor) e exiba:
//a) Quantos votos obteve cada sabor. b) Qual o sabor favorito segundo a pesquisa.

fun main() {
    println("Pesquisa SABOR FAVORITO!\n"
            + "Qual o seu sabor favorito de pizza? \n"
            + "       Sabores  | Código \n"
            + "     Mussarela  |    5   \n"
            + "     Calabresa  |   25   \n"
            + "Quatro Queijos  |   50   \n")

    var M5 = 0
    var C25 = 0
    var Q50 = 0
    var i = 0

    while (i < 10) {
        val random = Random.nextInt(0, 3)

        when (random) {
            0 -> M5++
            1 -> C25++
            2 -> Q50++
        }
        i++
        println("-------------------------------\n" +
                 i +"ª Pessoa votou. \n" +
                "Total de votos: $i \n" +
                "Mussarela: $M5 \n" +
                "Calabresa: $C25 \n" +
                "Quatro-Queijos: $Q50\n")
    }

    print("O sabor preferido é ")

    if(M5 > C25 && M5 > Q50){
        println("Mussarela!")
    }else if(C25 > M5 && C25 > Q50){
        println("Calabresa!")
    }else{
        println("Quatro-Queijos!")
    }


}